README.md

```diff
@@ -0,0 +1,52 @@
+# Shell Script
+
+This script (`bogons_sync-ripe.sh`) downloads the nro-delegated-stats file from RIPE and retrieves the IPv4/IPv6 prefixes and ASNs that are either available or reserved.
+Furthermore, it applies them to bird's default configuration file (bird.conf) as filters.
+
+## Prerequisites
+
+1. bc - An arbitrary precision calculator language. For Ubuntu: `apt-get install bc` 
+
+2. iprange - manage IP ranges. For Ubuntu: `apt-get install iprange` 
+
+## Use
+
+The script without any arguments (e.g. `./bogons-sync-ripe.sh`), outputs 4 files: ASN numbers, and IPv4/IPv6 prefixes in various formats, allowing the user to feed the script's output in the routing daemon of choice.
+
+If the script is run with parameter the config file of bird (e.g. `./bogons-sync-ripe.sh bird.conf`),
+it applies the outpout in bird's default configuration file in the form of filters.
+
+## Cron
+
+A cronjob that runs every hour, with script outputs appended to a log file):
+
+```
+0 * * * * /root/bogons-sync-ripe/bogons_sync-ripe.sh bird.conf | tee -a /root/bogons-sync-ripe/output.log 2>&1 2>>/root/bogons-sync-ripe/bogons_sync-ripe.err 1>/dev/null
+```
+
+# Bird's configuration file
+
+This file (`bird.conf`) can be used as guide in order to configure your Route Collector, if you choose to use bird 2 as your routing daemon.
+
+## Note:
+Because of the thousands of prefixes and ASNs are generated by the bash script and in order for all of them to set as bird's filters, the lines in bird.conf which contain the ASN numbers as well as the IPV4/IPV4 prefixes have to be between commented marks, e.g.:
```

Comment:
this is wrong.
my original comment was "In order for these lines to be applied correctly" has to be here. Το πρόβλημα που προκύπτει είναι το editing του αρχείου με τα standard commands (π.χ sed), δεν είναι το μέγεθος των strings. Εδώ πρέπει να περιγραφεί ότι ο λόγος των marks είναι για να μπορεί να γίνει σωστά και γρήγορα το "editing" του bird.conf από το script.

πώς θα αποτυπωθεί αυτό καλύτερα στα αγγλικά?

Reply:
Fixed